pkg/test/framework/runtime/components/environment/kube/environment.go

```diff
@@ -168,11 +170,7 @@ func (e *Environment) MinikubeIngress() bool {
 
 // HelmValueMap returns the overrides for helm values.
 func (e *Environment) HelmValueMap() map[string]string {
-	out := make(map[string]string)
-	for k, v := range e.s.Values {
-		out[k] = v
-	}
-	return out
+	return e.s.Values
```

Comment:
why change this?

Reply:
I changed this before because I want to make `helmValuesByTest map[lifecycle.Scope]map[string]string{}`, and synthesize from that map on demand, but it turns out not very helpful. As aforementioned, istio control plane is always system scope, started in kube environment...

Yeah, we can remove this given now. Will Do.